docs/SAFETY.md

```diff
@@ -0,0 +1,26 @@
+## Safety
+
+![Warning](imgs/manual/warning.png)
+
+- **WARNING!** TO REDUCE THE RISK OF FIRE OR ELECTRIC SHOCK, DO NOT EXPOSE THIS APPLIANCE TO RAIN OR MOISTURE.
+- **CAUTION!** TO REDUCE THE RISK OF ELECTRIC SHOCK, KEEP COVER ON AT ALL TIMES.
+
+### IMPORTANT SAFETY INSTRUCTIONS
+- Read and keep these instructions.
+- Heed all warnings and follow all instructions contained within this manual.
+- Do not use this unit near water.
+- Clean only with dry cloth.
+- When cleaning, clean exterior only
+- Do not block any ventilation openings. Install in accordance with the manufacturer’s instructions.
+- Do not install near any heat sources such as radiators, heat registers, stoves, or other apparatus (including amplifiers) that produce heat.
+- Do not defeat the safety purpose of the polarized or grounding-type plug. A polarized plug has two blades with one wider than the other. A grounding type plug has two blades and a third grounding prong. The wide blade or the third prong are provided for your safety. If the provided plug does not fit into your outlet, consult an electrician for replacement of the obsolete outlet.
+- Protect the power cord from being walked on or pinched particularly at plugs, convenience receptacles, and the point where they exit from the unit.
+- Only use attachments/accessories specified by the manufacturer.
+- Unplug this unit during lightning storms or when unused for long periods of time. **EDIT EDIT EDIT**
```

Comment:
edit

Reply:
Ah, forgot to mention but the **EDIT EDIT EDIT** sections were just the bits I was unsure about as I was editing the original draft, there will be a few of those around and they're essentially just requests for opinion/clarification

Reply:
what clarification were you seeking here?

Reply:
Personally, I don't think there's a ton of risk in leaving it as-is. It feels excessive, but there are perhaps reasons for it being written this way.